app/lib/bk/compat/parsers/bitrise.rb

```diff
@@ -41,12 +41,29 @@ def parse
 
         stack_value = @config.dig('meta', 'bitrise.io', 'stack')
         agents = stack_value ? { stack: stack_value } : {}
+
+        env = process_app_envs
+
         Pipeline.new(
           steps: bk_steps,
-          agents: agents
+          agents: agents,
+          env: env
```

Comment:
why not call the method directly instead of creating an intermediate variable?

```suggestion
        Pipeline.new(
          steps: bk_steps,
          agents: agents,
          env: process_app_envs
```